src/style-spec/package.json

```diff
@@ -9,14 +9,23 @@
     "mapbox-gl-js"
   ],
   "license": "ISC",
-  "main": "./dist/index.js",
+  "main": "./dist/index.cjs",
   "module": "./dist/index.es.js",
   "type": "module",
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.es.js"
+    },
+    "./": {
+      "import": "./"
```

Comment:
Note: Unlike esm, the presence of `"exports"` apparently does not affect which files `require` is allowed to require, so I've not explicitly exported files for require (also, I'm not aware this pattern is even in usage, based on a [search of the mapbox org](https://github.com/search?l=JavaScript&p=1&q=org%3Amapbox+mapbox-gl-style-spec&type=Code))